demo/button/button-icon.html

```diff
@@ -60,7 +60,7 @@
 		}
 	</style>
 </head>
-<body class="d2l-typography" unresolved="unresolved">
+<body class="d2l-typography">
```

Comment:
I had to remove all these as with Polymer excluded they're no longer getting automatically removed. We'll need to do this differently if we feel like it's valuable.

Reply:
Right...  I think without it I was seeing some flash of unwanted text due to the font loading.

Reply:
There's definitely some FOUC/FOUT/flash-of-unupgraded-web-components. I think we want to fix it somehow, just not in this PR. :)